src/Scripting/Core/ObjectFormatter.cs

```diff
@@ -27,53 +29,64 @@ public string FormatObject(object obj, ObjectFormattingOptions options = null)
 
         #region Reflection Helpers
 
-        private static bool HasOverriddenToString(Type type)
+        private static bool HasOverriddenToString(TypeInfo type)
         {
-            Ref.MethodInfo method = type.GetMethod(
```

Comment:
GetDeclaredMethod is actually in ReflectionUtils. Maybe we need to do some more unification here.